examples/api/test/material/scaffold/scaffold_messenger_state.show_snack_bar.1_test.dart

```diff
@@ -0,0 +1,24 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/material/scaffold/scaffold_messenger_state.show_snack_bar.1.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('Floating SnackBar is visible', (WidgetTester tester) async {
```

Comment:
If you add a fail state to the example, you'll want to verify the error is thrown in a test here.

Reply:
Done